.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Report something that is broken or incorrect
+labels: bug
+---
+
+<!--
+# nf-core/tools bug report
+
+Hi there!
+
+Thanks for telling us about a problem with the nf-core/tools package.
+Please delete this text and anything that's not relevant from the template below:
+-->
+
+## Description of the bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## Steps to reproduce
+
+Steps to reproduce the behaviour:
+
+1. Command line: <!-- [e.g. `nf-core lint ...`] -->
+2. See error: <!-- [Please provide your error message] -->
+
+## Expected behaviour
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+## System
+
+- Hardware: <!-- [e.g. HPC, Desktop, Cloud...] -->
+- Executor: <!-- [e.g. slurm, local, awsbatch...] -->
+- OS: <!-- [e.g. CentOS Linux, macOS, Linux Mint...] -->
+- Version <!-- [e.g. 7, 10.13.6, 18.3...] -->
```

Comment:
Maybe we should add the Python version? This is the template for Tools after all and not for the pipelines.